enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/ProcedureInteractionTestBase.java

```diff
@@ -357,8 +335,14 @@ void assertPasswordChangeWhenPasswordChangeRequired( S subject, String newPasswo
     void assertFail( S subject, String call, String partOfErrorMsg )
     {
         String err = assertCallEmpty( subject, call );
-        assertThat( err, not( equalTo( "" ) ) );
-        assertThat( err, containsString( partOfErrorMsg ) );
+        if ( partOfErrorMsg == null || partOfErrorMsg == "" )
+        {
+            assertThat( err, not( equalTo( "" ) ) );
```

Comment:
and since we know it's null or empty here, why just not to use `nullValue` here instead of negation?